### PR TITLE
CFU Title Caps & scanner/scan rule standardization

### PR DESCRIPTION
**Pull request by kingthorin**
Follow-up to: zaproxy/zaproxy#7708, related to: zaproxy/zaproxy#2000 & zaproxy/zaproxy#4732.

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Comment by kingthorin** [comment]
Should `cfu.kali.options` be de-HTML'ified? (Per: #7521, #7538, #7620, etc.)

File: zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties [hunk 1]
```diff
@@ -1119,19 +1119,19 @@ cfu.options.dialog.dirs.remove.button.confirm	= Remove
...
-cfu.options.downloadNewRelease 		= Automatically download new ZAP releases (you will be prompted to install them)
-cfu.options.downloadNewRelease.kali = Automatically download new ZAP releases (disabled on Kali)
-cfu.options.checkAddonUpdates		= Check for updates to the add-ons you have installed
-cfu.options.installAddonUpdates		= Automatically install updates to the add-ons you have installed
-cfu.options.installScannerRules		= Automatically install updates to the scanner rules you have installed
+cfu.options.downloadNewRelease 		= Automatically download new ZAP releases (you will be prompted to install them).
+cfu.options.downloadNewRelease.kali = Automatically download new ZAP releases (disabled on Kali).
+cfu.options.checkAddonUpdates		= Check for updates to the add-ons you have installed.
+cfu.options.installAddonUpdates		= Automatically install updates to the add-ons you have installed.
+cfu.options.installScannerRules		= Automatically install updates to the scan rules you have installed.
```
Comment (thc202) [inline zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties:1126]: In the installer it's using `scanner`:
https://github.com/zaproxy/zaproxy/blob/430d52792b7da1e57492f8deb21d667ff08b1f0c/zap/src/main/installer/resources/Messages.properties#L15

Comment (kingthorin) [reply zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties:1126]: Are we in agreement that "scan rule" is better? If so I'll tweak the installer set too.

**Comment by thc202** [comment]
https://github.com/zaproxy/zaproxy/pull/7722#issuecomment-1416492415

No, that uses `ZapHtmlLabel`.

**Comment by kingthorin** [comment]
Thanks

**Comment by kingthorin** [comment]
I’ll look at doing a core help PR too.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**